crates/router/src/compatibility/stripe/app.rs

```diff
@@ -37,10 +48,12 @@ impl Refunds {
     pub fn server(config: routes::AppState) -> Scope {
         web::scope("/refunds")
             .app_data(web::Data::new(config))
-            .service(refund_create)
-            .service(refund_retrieve)
-            .service(refund_update)
-            .service(refund_retrieve_with_gateway_creds)
+            .service(web::resource("/").route(web::post().to(refund_create)))
+            .service(web::resource("/{refund_id}").route(web::get().to(refund_retrieve)))
+            .service(web::resource("/{refund_id}").route(web::post().to(refund_update)))
+            .service(
+                web::resource("/sync").route(web::post().to(refund_retrieve_with_gateway_creds)),
```

Comment:
The `sync` route would match to `refund_update` which is above it so better to move this path to the top of refunds.